### PR TITLE
Update int gem mods lines 1486-3453

### PR DESCRIPTION
**Pull request by markoleptic**
Updated int gem mods IceNovaAltX, InceNovaAltY, IceSpear, IceSpearAltX, ColdProjectileMineAltX, ColdProjectileMineAltY, KineticBoltAltX, LightningTendrilsChannelledAltX, LightningTendrilsChannelledAltY, MagmaSigilAltX,
MagmaSigilAltY, PowerSiphonAltX, SanctifyAltX, MortarBarrageMine, MortarBarrageMineAltX.

Skipped: Lightning Spire Trap, Lightning Spire Trap of Zapping, Penance Brand variants, and various other mods I didn't know how to implement.

### CODE REVIEW

File: src/Export/Skills/act_int.txt [hunk 1]
```diff
@@ -2515,10 +2630,52 @@ local skills, mod, flag, skill = ...
...
+		{
+			name = "First pulse",
+		},
+		{
+			name = "After first pulse",
+		},
+	},
+	preDamageFunc = function(activeSkill, output)
+		activeSkill.skillData.hitTimeOverride = activeSkill.skillData.repeatFrequency / (1 + activeSkill.skillModList:Sum("INC", activeSkill.skillCfg, "Speed", "BrandActivationFrequency") / 100) / activeSkill.skillModList:More(activeSkill.skillCfg, "BrandActivationFrequency")
+	end,
```
Comment (LocalIdentity) [inline src/Export/Skills/act_int.txt:2643]: preDamage func goes above skill parts

File: src/Export/Skills/act_int.txt [hunk 2]
```diff
@@ -2515,10 +2630,52 @@ local skills, mod, flag, skill = ...
...
```
Comment (LocalIdentity) [inline src/Export/Skills/act_int.txt:2673]: move above skill parts

File: src/Export/spec.lua [hunk 3]
```diff
@@ -4516,15 +4516,6 @@ return {
...
-	genericbuffauras={
-		[1]={
-			list=false,
-			name="Id",
-			refTo="",
-			type="String",
-			width=150
-		}
-	},
 	genericleaguerewardtypes={
```
Comment (LocalIdentity) [inline src/Export/spec.lua:4519]: remove this file from the PR

File: src/Modules/ConfigOptions.lua [hunk 4]
```diff
@@ -417,6 +417,14 @@ return {
...
 		modList:NewMod("Condition:CastOnFrostbolt", "FLAG", true, "Config", { type = "SkillName", skillName = "Ice Nova" })
 	end },
+    { label = "", ifSkill = "Ice Nova of Frostbolts" },
+	{ var = "iceNovaOfFrostboltsCastOnFrostbolt", type = "check", label = "Cast on Frostbolt?", ifSkill = "Ice Nova of Frostbolts", apply = function(val, modList, enemyModList)
+		modList:NewMod("Condition:CastOnFrostbolt", "FLAG", true, "Config", { type = "SkillName", skillName = "Ice Nova of Frostbolts" })
+	end },
+    { label = "Ice Nova of Deep Freeze:", ifSkill = "Ice Nova of Deep Freeze" },
+	{ var = "iceNovaOfDeepFreezeCastOnFrostbolt", type = "check", label = "Cast on Frostbolt?", ifSkill = "Ice Nova of Deep Freeze", apply = function(val, modList, enemyModList)
+		modList:NewMod("Condition:CastOnFrostbolt", "FLAG", true, "Config", { type = "SkillName", skillName = "Ice Nova of Deep Freeze" })
+	end },
```
Comment (LocalIdentity) [inline src/Modules/ConfigOptions.lua:427]: should all be using the existing cast on Frostbolts config

File: src/Export/Skills/act_int.txt [hunk 5]
```diff
@@ -2623,16 +2812,16 @@ local skills, mod, flag, skill = ...
...
 		},
 		["mortar_barrage_mine_minimum_added_fire_damage_taken_limit"] = {
-			mod("Multiplier:PyroclastSelfFireMinLimit", "BASE", nil, 0, 0, { type = "GlobalEffect", effectType = "AuraDebuff", unscalable = true, effectName = "Pyroclast Mine Limit" }),
+			mod("Multiplier:PyroclastSelfFireMinLimit", "BASE", nil, 0, 0, { type = "SkillName", skillName = "Pyroclast Mine" }, { type = "GlobalEffect", effectType = "AuraDebuff", unscalable = true, effectName = "Pyroclast Mine Limit" }),
 		},
 		["mortar_barrage_mine_maximum_added_fire_damage_taken_limit"] = {
-			mod("Multiplier:PyroclastSelfFireMaxLimit", "BASE", nil, 0, 0, { type = "GlobalEffect", effectType = "AuraDebuff", unscalable = true, effectName = "Pyroclast Mine Limit" }),
+			mod("Multiplier:PyroclastSelfFireMaxLimit", "BASE", nil, 0, 0, { type = "SkillName", skillName = "Pyroclast Mine" }, { type = "GlobalEffect", effectType = "AuraDebuff", unscalable = true, effectName = "Pyroclast Mine Limit" }),
```
Comment (LocalIdentity) [inline src/Export/Skills/act_int.txt:2824]: Have a look at the comments @Regisle made in Discord

File: src/Export/Skills/act_int.txt [hunk 6]
```diff
@@ -2535,6 +2692,18 @@ local skills, mod, flag, skill = ...
...
+	statMap = {
+		["critical_strike_chance_+%_final_per_power_charge_from_power_siphon"] = {
+			mod("CritChance", "MORE", nil, 0, 0, { type = "Multiplier", var = "PowerCharge" }),
+		},
+		["power_siphon_base_fire_at_x_targets"] = {
+			flag("OneShotProj")
+		},
+		["attack_skills_have_added_lightning_damage_equal_to_%_of_maximum_mana"] = {
+			mod("LightningMin", "BASE", nil, nil, ModFlag.Attack, { type = "PercentStat", stat = "Mana", percent = 1 }),
+			mod("LightningMax", "BASE", nil, nil, ModFlag.Attack, { type = "PercentStat", stat = "Mana", percent = 1 }),
```
Comment (LocalIdentity) [inline src/Export/Skills/act_int.txt:2704]: Should not be nil,nil. We showed on Discord what the mod should look like

File: src/Export/Skills/act_int.txt [hunk 7]
```diff
@@ -1937,6 +1950,31 @@ local skills, mod, flag, skill = ...
...
+	statMap = {
+		["ice_spear_second_form_projectile_speed_+%_final"] = {
+			mod("ProjectileSpeed", "MORE", nil, 0, 0, { type = "SkillPart", skillPartList = { 2, 4 } }),
+		},
+	},
+	preDamageFunc = function(activeSkill, output)
+		if activeSkill.skillPart == 3 or activeSkill.skillPart == 4 then
+			activeSkill.skillData.dpsMultiplier = (activeSkill.skillData.dpsMultiplier or 1) * output.ProjectileCount
+		end
+	end,
```
Comment (LocalIdentity) [inline src/Export/Skills/act_int.txt:1976]: func should be above parts

**Review by LocalIdentity — CHANGES_REQUESTED** [review]

File: src/Export/Skills/act_int.txt (hunk 5 above)
Comment (markoleptic) [reply src/Export/Skills/act_int.txt:2827]: Reverted all new names to same as base gem, but kept the skillName checks for both versions. Works properly in PoB.

**Review by LocalIdentity — APPROVED** [review]

**Pull request merged**